src/main.ts

```diff
@@ -1216,11 +1218,7 @@ export default class ObsidianGit extends Plugin {
 I strongly recommend to use "Source mode" for viewing the conflicted files. For simple conflicts, in each file listed above replace every occurrence of the following text blocks with the desired text.
 
 \`\`\`diff
-<<<<<<< HEAD
```

Comment:
This should not be deleted, it may look like a merge conflict, but is meant to be this way.